src/status_im2/common/resources.cljs

```diff
@@ -74,6 +74,16 @@
    :verified             (js/require "../resources/images/mock2/verified.png")
    :user-picture-female2 (js/require "../resources/images/mock2/user_picture_female2.png")
    :user-picture-male4   (js/require "../resources/images/mock2/user_picture_male4.png")
+   :ethereum-logo        (js/require "../resources/images/mock2/Ethereum.png")
+   :optimism-logo        (js/require "../resources/images/mock2/Optimism.png")
+   :arbitrum-logo        (js/require "../resources/images/mock2/Arbitrum.png")
+   :zkSync-logo          (js/require "../resources/images/mock2/zkSync.png")
+   :hermez-logo          (js/require "../resources/images/mock2/Hermez.png")
+   :xDAI-logo            (js/require "../resources/images/mock2/xDAI.png")
+   :polygon-logo         (js/require "../resources/images/mock2/Polygon.png")
+   :scroll-logo          (js/require "../resources/images/mock2/Scroll.png")
+   :taiko-logo           (js/require "../resources/images/mock2/Taiko.png")
+   :unknown-logo         (js/require "../resources/images/mock2/Unknown.png")
    :user-picture-male5   (js/require "../resources/images/mock2/user_picture_male5.png")})
```

Comment:
We need those under `quo2.foundations.resources`

Reply:
@OmarBasem should me make a `get-tokens` function in the resources file for quo2 to separate them somewhat?

Reply:
@J-Son89 we now have a bunch of tokens (and networks) under `quo2.foundations.resources`. I will try to make a PR to add the rest of the tokens and remove any duplications from `status_im2`

Reply:
Nice one @OmarBasem!

Reply:
Done!